src/MerQure.RbMQ/Config/MerQureConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace MerQure.RbMQ.Config
+{
+    public class MerQureConfiguration
+    {
+        private const string DefaultPrefetchCountPropertyName = "defaultPrefetchCount";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'DefaultPrefetchCountPropertyName'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1144)